external/fv3fit/fv3fit/pytorch/graph_builder.py

```diff
@@ -0,0 +1,46 @@
+import torch
+import numpy as np
+import geopy.distance
+from vcm.catalog import catalog
+import dataclasses
+
+
+@dataclasses.dataclass
+class GraphConfig:
+    """
+    Attributes:
+        neighbor: number of nearest neighbor grids
+        coarsen: 1 if the full model resolution is used, othewise data will be coarsen
+        resolution: Model resolution to load the corresponding lat and lon.
+    """
+
+    neighbor: int = 10
+    coarsen: int = 8
+    resolution: str = "grid/c48"
+
+
+def build_graph(config: GraphConfig) -> tuple:
```

Comment:
For tuple typing you should use e.g. `Tuple[torch.Tensor, torch.Tensor]` (not sure those are the correct pytorch types in this case) . Can be imported using `from typing import Tuple`.